src/cc2olx/cli.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+
+from pathlib import Path
+
+RESULT_TYPE_FOLDER = "folder"
+RESULT_TYPE_ZIP = "zip"
+
+
+def parse_args(args=None):
+    parser = argparse.ArgumentParser(
+        description="This script converts imscc files into folders with "
+        "all the content; in the defined folder structure."
+    )
+    parser.add_argument(
+        "-i",
+        "--inputs",
+        nargs="*",
+        type=lambda p: Path(p).absolute(),
+        required=True,
+        help="Please provide the paths to the imscc files or directories "
+        "that contain them.",
+    )
+    parser.add_argument(
+        "-ll",
```

Comment:
By convention, at least with GNU/linux systems (and definitely everywhere in the edx ecosystem), short args are only a single letter; can we make this `-l`?